MovieApp/PresentationLayer/HomeScreen/CategoryCollection/CategoryCollectionView+Design.swift

```diff
@@ -0,0 +1,59 @@
+import UIKit
+
+extension CategoryCollectionView: ConstructViewsProtocol {
+
+    func buildViews() {
+        createViews()
+        styleViews()
+    }
+
+    func createViews() {
+        title = UILabel()
+        addSubview(title)
+
+        options = OptionBar()
+        addSubview(options)
+        bringSubviewToFront(options)
+
+        filmCollection = makeCollectionView()
+        addSubview(filmCollection)
+    }
+
+    func styleViews() {
+        title.text = "What's popular"
+        title.font = UIFont(name: "ProximaNova-Bold", size: 20)
+        title.textColor = UIColor(named: "DarkBlue")
+
+        filmCollection.showsHorizontalScrollIndicator = false
+    }
+
+    private func makeCollectionView() -> UICollectionView {
+        let layout = UICollectionViewFlowLayout()
+        layout.scrollDirection = .horizontal
+
+        let collectionView = UICollectionView(frame: .zero, collectionViewLayout: layout)
+        collectionView.backgroundColor = .white
+        collectionView.register(MoviePosterCell.self, forCellWithReuseIdentifier: MoviePosterCell.cellIdentifier)
+        collectionView.dataSource = self
+
+        return collectionView
+    }
+
+    func defineLayoutForViews() {
+        title.snp.makeConstraints {
+            $0.top.leading.trailing.equalToSuperview().inset(defaultInset)
+        }
+
+        options.snp.makeConstraints {
+            $0.top.equalTo(title.snp.bottom).offset(defaultSpacing)
+            $0.leading.trailing.equalToSuperview().inset(defaultInset)
+        }
+
+        filmCollection.snp.makeConstraints {
```

Comment:
pin filmCollection bottom to superview bottom. It has a height so it will show but it might not once you put it in stack view...

Reply:
When I pin it to the superview's bottom the spacing between the film collection and the option bar becomes too large.
![Simulator Screen Shot - iPod touch (7th generation) - 2021-07-23 at 09 43 18](https://user-images.githubusercontent.com/53893982/126751732-c6faf9cd-5fb9-423f-968c-3b946eda0519.png)